### ISSUE TITLE
Use MSA pairformer to obtain logits for given position in sequence

### ISSUE THREAD

**Issue by charlieccp** [yoakiyama/MSA_Pairformer#17]
Hi, it's unclear to me if I can use MSA pairformer to obtain per position logits, for example to suggest which residue is more likely at a given position given the context of the rest of the sequence. If yes, could you please help me figuring out how to obtain these?

**Comment by charlieccp** [comment]
For example with ESM2, I would use the following:

  from esm import pretrained

  model, alphabet = pretrained.esm2_t33_650M_UR50D()
  model = model.to(device).eval()
  batch_converter = alphabet.get_batch_converter()

  aa_tokens = list(alphabet.all_toks) 
  aa_to_idx = {tok: i for i, tok in enumerate(aa_tokens)}
....
